modules/apps/transfer/ibc_module_test.go

```diff
@@ -376,7 +376,7 @@ func (suite *TransferTestSuite) TestOnRecvPacket() {
 			}
 
 			seq := uint64(1)
-			packet = channeltypes.NewPacket(packetData.GetBytes(), seq, path.EndpointA.ChannelConfig.PortID, path.EndpointA.ChannelID, path.EndpointB.ChannelConfig.PortID, path.EndpointB.ChannelID, clienttypes.NewHeight(1, 100), 0)
+			packet = channeltypes.NewPacket(packetData.GetBytes(), seq, path.EndpointA.ChannelConfig.PortID, path.EndpointA.ChannelID, path.EndpointB.ChannelConfig.PortID, path.EndpointB.ChannelID, clienttypes.ZeroHeight(), uint64(suite.chainA.GetContext().BlockTime().UnixNano())+ibctesting.DefaultTimeoutTimestampDelta)
```

Comment:
Could it be an idea to create a function to get this value? With all the uses in relay_forwarding_test as well it might make things a bit more tidy?

Reply:
Sounds good, @gjermundgaraba. If you have something concrete in mind, feel free to push a commit. Otherwise I will merge the PR tonight.

Reply:
I got it

Reply:
Thank you, @colin-axner!